.github/_typos.toml

```diff
@@ -1,7 +1,10 @@
 [files]
 extend-exclude = [
 	"*.js*", "*.json", "*.svg",
-	"*signed.txt", "signed_disclosure.txt", # can be removed once https://github.com/crate-ci/typos/issues/401 is fixed
+	"/unstyled_docs",
```

Comment:
There are a ton of typos in the old spec with some relating to identifiers. At least for now I have just ignored all of the `unstyled_docs`.